auth/store.go

```diff
@@ -997,18 +997,23 @@ func (as *authStore) AuthInfoFromCtx(ctx context.Context) (*AuthInfo, error) {
 		return nil, nil
 	}
 
-	ts, tok := md["token"]
-	if !tok {
-		return nil, nil
-	}
-
-	token := ts[0]
-	authInfo, uok := as.authInfoFromToken(ctx, token)
-	if !uok {
-		plog.Warningf("invalid auth token: %s", token)
-		return nil, ErrInvalidAuthToken
+	for k := range md {
+		if k == "token" || k == "authorization" {
+			ts, tok := md[k]
+			if !tok {
+				return nil, nil
+			}
+			token := ts[0]
+			authInfo, uok := as.authInfoFromToken(ctx, token)
+			if !uok {
+				plog.Warningf("invalid auth token: %s", token)
+				return nil, ErrInvalidAuthToken
+			}
+			plog.Debugf("checking value in loop %s", k)
```

Comment:
if we do want the debugging logging here, it needs to contain more context. checking value in a loop is too vague.